client/components/theme-tier/constants.js

```diff
@@ -22,11 +22,11 @@ export const THEME_TIERS = {
 		minimumUpsellPlan: PLAN_FREE,
 	},
 	personal: {
-		label: translate( 'Personal' ),
+		label: translate( 'Starter' ),
```

Comment:
> For greater coherence the theme tiers will be named after the minimum plan level required to use them.

In that case, would something like this perhaps be preferable for consistency with how the rest of Calypso handles plan names?

```suggestion
		label: getPlan( PLAN_PERSONAL )?.getTitle(),
```